src/app/dynamic/enterprise/quotas/project-quota-dialog/template.html

```diff
@@ -65,32 +69,45 @@
       </mat-error>
     </form>
 
-    <div fxLayout="column"
+    <div *ngIf="!editQuota"
+         fxLayout="column"
          fxLayoutAlign=" stretch">
-      <mat-form-field class="km-dropdown-with-suffix"
-                      *ngIf="!editQuota; else editMode">
+      <mat-form-field fxFlex
+                      *ngIf="projectControl as control">
         <mat-label>Project</mat-label>
-        <mat-select disableOptionCentering
-                    class="km-select-ellipsis"
-                    [formControl]="form.controls.subjectName"
-                    required>
-          <mat-option *ngFor="let project of projects"
-                      [value]="project.id"> {{project.name}} </mat-option>
-          <mat-select-trigger fxLayout="row">
-            <div fxLayoutAlign="start">{{selectedProject?.name || editQuota?.name}}</div>
+        <input matInput
+               type="text"
+               [formControl]="control"
+               [matAutocomplete]="autoComplete" />
+        <mat-spinner matSuffix
+                     *ngIf="!projects.length"
+                     [diameter]="20"></mat-spinner>
+        <mat-autocomplete autoActiveFirstOption
+                          #autoComplete="matAutocomplete"
+                          [displayWith]="projectDisplayFn">
+          <mat-option *ngFor="let project of projects | filterBy: { name: control.value?.name ? control.value.name : control.value }"
+                      [value]="project">
+            <div fxLayout="row"
+                 fxLayoutGap="10px"
+                 fxLayoutAlign=" center">
+              <span>{{project.name}}</span>
 
-            <div fxLayoutAlign="end"
-                 class="km-select-trigger-button-wrapper">
-              <button matSuffix
-                      mat-icon-button
-                      [disabled]="!!editQuota"
-                      (click)="resetSubjectNameControl()">
-                <i class="km-icon-mask km-icon-remove"></i>
-              </button>
+              <mat-card-subtitle class="km-no-margin"
```

Comment:
```suggestion
              <mat-card-subtitle class="km-no-margin km-no-padding"
```
So that it is aligned with project name.